Resources/doc/reference/getting_started.rst

```diff
@@ -99,11 +99,53 @@ Render article fragments
 ------------------------
 
 **This part is currently in development.**
-The goal is to provide a simple twig helper which will render each fragments associated to an article.
-Each fragment has its own template, this twig function is going to need to fetch in ``FragmentService`` the template,
-then render it.
+SonataArticleBundle now comes with a twig helper which allows you to render article's fragments
+if they are enable.
 
 
 .. code-block:: jinja
 
-    {{ sonata_article_render_fragments(article) }}
+    {{ sonata_article_render_article_fragments(article) }}
```

Comment:
maybe `sonata_article_render_fragments` would be unique enough? Especially since `article` is passed as argument.